### DISCUSSION TITLE
Go back

### DISCUSSION THREAD

**Discussion by loan-talvat** [AzuraCast/AzuraCast#5614] [Ideas]
Being able to go back, for example, 20 minutes back from the stream (at the start of a program or other..)

**Comment by Mozitux** [comment]
I'm not sure what you really want to do, but if you want listeners to be able to pause the live and rewind during a live broadcast, you will need to implement this feature yourself or make a feature proposal. 

On the other hand, do not expect anything immediately, currently the development of Azuracast is slower than usual due to the personal difficulties of the lead developer.
